jetty-plus/src/main/java/org/eclipse/jetty/plus/annotation/LifeCycleCallback.java

```diff
@@ -76,12 +78,16 @@ public Method getTarget()
 
     public void setTarget (String className, String methodName)
     {
+        Objects.requireNonNull(className);
+        Objects.requireNonNull(methodName);
```

Comment:
Can't we use constructor injection, with `final` fields, etc.

Reply:
Done.